authui/src/authflowv2/base.css

```diff
@@ -30,31 +30,40 @@
 
   /* Colors */
   :root {
-    --color-white: #ffffff;
-    --color-black: #000000;
     --color-success: #11c624;
     --color-error: #ff295b;
     --color-warning: #f39d17;
 
+    /* Primary colors is generated using fluent ui theme generator */
+    /* https://developer.microsoft.com/en-us/fluentui?fabricVer=5#/styles/themegenerator */
+    /* Parameters:
+      Primary color: #176df3
+    */
+
     /* primary */
-    --color-primary-base: #176df3;
-    --color-primary-100: #e2e7fe;
-    --color-primary-200: #adbdfd;
-    --color-primary-300: #7295fc;
-    --color-primary-400: #176df3;
-    --color-primary-500: #0d4cae;
-    --color-primary-600: #052e6f;
-    --color-primary-700: #011234;
-
-    /* grey */
-    --color-grey-base: #73747b;
-    --color-grey-100: #e7e7e9;
-    --color-grey-200: #bfbfc3;
-    --color-grey-300: #98999e;
-    --color-grey-400: #73747b;
-    --color-grey-500: #505157;
-    --color-grey-600: #303135;
-    --color-grey-700: #131315;
+    --color-primary-theme-darker: #0c3c88;
+    --color-primary-theme-dark: #1151b8;
+    --color-primary-theme-dark-alt: #1460da;
+    --color-primary-theme-primary: #176df3;
+    --color-primary-theme-secondary: #317bf4;
+    --color-primary-theme-tertiary: #70a4f7;
+    --color-primary-theme-light: #b7d1fb;
+    --color-primary-theme-lighter: #d8e6fd;
+    --color-primary-theme-lighter-alt: #f5f9fe;
+
+    /* Text colors is tonal palette */
+    /* https://m3.material.io/styles/color/system/how-the-system-works#395813c8-b314-48d1-bb55-266f421eb3a4 */
+    /* text */
+    --color-text-100: #e7e7e9;
```

Comment:
I've renamed grey to text color in this pr, because I think the role of the color is to display texts.

Reply:
@louischan-oursky Yes, or maybe I should simply name it --color-secondary?

Reply:
https://m3.material.io/styles/color/system/how-the-system-works

Design system often names the color "Primary", "Secondary", and "Tertiary". But "Secondary" is not for displaying text. Material design calls it "Neutral". 

https://spectrum.adobe.com/page/color-system/ Spectrum calls it grey, but it also mentions it is neutral.

Reply:
Actually the problem here is foreground elements and background elements are using the same palette, which are usually separated.
Anyway, lets rename it to neutral first.